MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -356,8 +358,63 @@ MeshLib::Mesh* VtkMeshConverter::convertUnstructuredGrid(vtkUnstructuredGrid* gr
 		elements[i] = elem;
 	}
 
-	return new MeshLib::Mesh(mesh_name, nodes, elements);
+	MeshLib::Mesh* mesh = new MeshLib::Mesh(mesh_name, nodes, elements);
+	convertScalarArrays(*grid, *mesh);
+
+	return mesh;
+}
+
+void VtkMeshConverter::convertScalarArrays(vtkUnstructuredGrid &grid, MeshLib::Mesh &mesh)
```

Comment:
Unfortunately not. VTK has not much const-methods, i.e. `grid.GetPointData()` is a non-const method.